cli/azd/pkg/output/colors.go

```diff
@@ -95,7 +96,18 @@ func WithMarkdown(markdownText string) string {
 }
 
 // WithHyperlink wraps text with the colored hyperlink format escape sequence.
+// When stdout is not a terminal (e.g., in CI/CD pipelines like GitHub Actions),
+// it returns the plain URL without escape codes to avoid displaying raw ANSI sequences.
 func WithHyperlink(url string, text string) string {
+	// Check if stdout is a terminal
+	if !isatty.IsTerminal(os.Stdout.Fd()) {
```

Comment:
Please use the `console.IsTerminal(...)` func instead since this leverages the same `go-isatty` package but also bundles a couple other checks.